### PR TITLE
Fixes #205 Handle newlines within logs

### LINKED ISSUE #205: WebUI logs do not handle newline within message
### This is a
- [x] Bug report
- [ ] Feature request

### Current behaviour

If a message contains newline chanracters, It is still printed as a single line.

### Expected behaviour

If a message recieved from the script contains newline character, It should be splitted into multiple lines.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue: #205 
## How Has This Been Tested?

Test at: https://yaydoc6.herokuapp.com/

## Types of changes

- [x] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: backend/generator.js [hunk 1]
```diff
@@ -35,14 +36,20 @@ exports.executeScript = function (socket, formData) {
...
+    terminal  : false
+  }).on('line', function(data) {
+    console.log(data);
+    socket.emit('logs', {donePercent: (donePercent = donePercent + 4), data: data});
   });
 
-  process.stderr.on('data', function (data) {
-    console.log(data.toString());
-    socket.emit('err-logs', data.toString());
+  readline.createInterface({
```
Comment (pri22296) [inline backend/generator.js]: These are all very similar statements. would it be possible to instead have a single function and call it with appropriate params?

**Comment by imujjwal96** [comment]
@pri22296 Updated it

File: util/output.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+var exports = module.exports = {};
+
+var readline = require('readline');
+
+exports.lineOutput = function (socket, process, event, increment) {
+  var donePercent = 0;
+  readline.createInterface({
+    input     : process.stdout,
+    terminal  : false
```
Comment (pri22296) [inline util/output.js]: Reason for terminal being false?

Comment (imujjwal96) [inline util/output.js]: No particular reason. I found this solution at https://stackoverflow.com/a/20272799/4592183

**Comment by imujjwal96** [comment]
@sch00lb0y  Please review.

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by imujjwal96** [comment]
@mariobehling  Resolved merge conflicts.

**Pull request merged**